content/blog/2020-05-19-remote-roundup/index.markdown

```diff
@@ -0,0 +1,115 @@
+---
```

Comment:
Note: reviewing the .markdown instead of the Rmd because lines wrap readably on GitHub for Markdown.